bio_hansel/main.py

```diff
@@ -192,36 +194,36 @@ def main():
     scheme_subtype_counts = subtype_counts(scheme_fasta)
     logging.debug(args)
     subtyping_params = init_subtyping_params(args, scheme)
-    input_genomes, reads = collect_inputs(args)
-    if len(input_genomes) == 0 and len(reads) == 0:
+    input_contigs, input_reads = collect_inputs(args)
+    if len(input_contigs) == 0 and len(input_reads) == 0:
```

Comment:
Should this be an "or" statement, that if either of these are empty, then it raises the error? Or is it ok if one of them is empty, that's still ok?

Reply:
We're allowing the user to specify a mix of file types. They can specify contigs (FASTA) and reads (FASTQ) in the same analysis. If both lists are empty then no input whatsoever has been specified so we can't do anything except let the user know they haven't specified anything to analyze.